.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -20,19 +19,18 @@ These points need to be fulfilled for every PR.
   - [ ] reformat the code with `scripts/dev/reformat-all`
   - [ ] make all unit tests pass
   - [ ] fix all memleaks
+  - [ ] fix the CI itself (or rebase if already fixed)
```

Comment:
That's a good addition 😅

Reply:
Yes, but the sub-items here shouldn't be checkboxes IMO.

```suggestion
  - fix the CI itself (or rebase if already fixed)
```

Making them checkboxes suggests it's something that has to be done for every PR. But only "The buildservers are happy." is the check that must be done for all PRs.